taiga_halo2/src/circuit/vp_circuit.rs

```diff
@@ -776,6 +778,19 @@ macro_rules! vp_verifying_info_impl {
                 }
             }
 
+            fn verify_transparently(
+                &self,
+            ) -> Result<ValidityPredicatePublicInputs, TransactionError> {
+                use halo2_proofs::dev::MockProver;
```

Comment:
This executes the circuit transparently?

Reply:
Yes, it is usually used in tests. The `MockProver` checks the constraints transparently instead of creating a proof.